cmd/codebuild/reset/service.go

```diff
@@ -62,6 +63,7 @@ func (svc *service) config() *serviceConfig {
 		nukeTemplateDefault: common.RequireEnv("RESET_NUKE_TEMPLATE_DEFAULT"),
 		nukeTemplateBucket:  common.RequireEnv("RESET_NUKE_TEMPLATE_BUCKET"),
 		nukeTemplateKey:     common.RequireEnv("RESET_NUKE_TEMPLATE_KEY"),
+		regions:             strings.Split(common.RequireEnv("RESET_NUKE_REGIONS"), ","),
```

Comment:
I tried to convert this whole file to use the new ConfigBuild, but I couldn't figure it out.
@nathanagood maybe you can give me a hand with this sometime?